src/piratechain/piratechainTypes.ts

```diff
@@ -8,7 +8,18 @@ import {
   asValue,
   Cleaner
 } from 'cleaners'
-import type { ErrorEvent as PiratechainErrorEvent } from 'react-native-piratechain'
+import type {
+  Addresses,
+  BlockRange,
+  ConfirmedTransaction,
+  ErrorEvent as PiratechainErrorEvent,
+  InitializerConfig,
+  SpendInfo,
+  SpendSuccess,
+  StatusEvent,
+  UpdateEvent,
+  WalletBalance
+} from 'react-native-piratechain'
```

Comment:
I'm not able to reproduce these exports from `react-native-piratechain`. Is there a package.json change somewhere that's missing from this branch?

Reply:
Yes, to versions of rnzcash and rnpirate that don't exist yet